packages/medusa/src/repositories/shipping-option.ts

```diff
@@ -1,5 +1,18 @@
-import { EntityRepository, Repository } from "typeorm"
+import { EntityRepository, In, Repository } from "typeorm"
 import { ShippingOption } from "../models/shipping-option"
 
 @EntityRepository(ShippingOption)
-export class ShippingOptionRepository extends Repository<ShippingOption> {}
+export class ShippingOptionRepository extends Repository<ShippingOption> {
+  public async upsertShippingProfile(
+    ids: string[],
```

Comment:
nit: Make it explicit what we are upserting profiles with
```suggestion
    shippingOptionIds: string[],
```